src/router/handler.ts

```diff
@@ -0,0 +1,20 @@
+import {ServerResponse} from "http";
+import {errors, HttpStatusCode} from "../constants/constants";
+import {baseHeaders} from "./router";
+
+export const showNotFoundView = (res: ServerResponse, message = errors.notFound) => {
+    res.writeHead(HttpStatusCode.NOT_FOUND, {
+        "Content-Type": "text/html"
+    });
+    res.end(`<h1>Error: ${message}</h1>`)
+};
+export const responseHandlerHTML = (res: ServerResponse, statusCode: HttpStatusCode, message: string) => {
+    res.writeHead(statusCode, {
+        "Content-Type": "text/html"
+    });
+    res.end(`<h1>${message}</h1>`)
```

Comment:
It will work, but in your approach API consumer should understand two type of responses JSON and HTML -
The general approach is to send JSON as more convenient to work with data and the client side will show the UI as they will decide.... It's could be in canvas or in push message... So html - inconvenient.